client/src/app/domain/models/base/base-model.ts

```diff
@@ -45,7 +45,7 @@ export abstract class BaseModel<T = any> implements Identifiable, Deserializable
         const updateCopy = { ...update }; // To not modifying the original update
         for (const key of Object.keys(update)) {
             const value = update[key as keyof T] as unknown;
-            if (isTemplateField(key) && Array.isArray(value) && value.length === 0) {
+            if (isTemplateField(key) && Array.isArray(value) && value.length >= 0) {
```

Comment:
`value.length >= 0` should always be true, either remove it or replace it with a condition that does, what this is intended to do there

Reply:
Ups. You are right ;)
Removed it.